application/views/admin/global_settings/_bounce.php

```diff
@@ -5,69 +5,75 @@
  */
 ?>
 
+<div class="container-fluid">
+    <div class="ls-flex-column ls-space padding left-5 right-35 col-md-7">
+        <div class="form-group">
+            <label class=" control-label" for='siteadminbounce'><?php eT("Default site bounce email:"); ?></label>
+            <div class="">
+                <input class="form-control" type='text' size='50' id='siteadminbounce' name='siteadminbounce' value="<?php echo htmlspecialchars(getGlobalSetting('siteadminbounce')); ?>" />
```

Comment:
deprecated too in 4.0 : https://github.com/LimeSurvey/LimeSurvey/blob/301cb150fab348e55247111e58933ce7a0c5090d/application/helpers/globalsettings_helper.php#L34